frontend/src/components/pages/rp-connect/secrets/Secrets.List.tsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import {PageComponent, PageInitHelper} from '../../Page';
+import {observer} from 'mobx-react';
+import {appGlobal} from '../../../../state/appGlobal';
+import {rpcnSecretManagerApi} from '../../../../state/backendApi';
+import {Features} from '../../../../state/supportedFeatures';
+import {Box, Button, ButtonGroup, Code, ConfirmItemDeleteModal, CopyButton, createStandaloneToast, DataTable, Flex, Image, SearchField, Text, Tooltip} from '@redpanda-data/ui';
+import Section from '../../../misc/Section';
+import PageContent from '../../../misc/PageContent';
+import {uiSettings} from '../../../../state/ui';
+import {Link} from 'react-router-dom';
+import {PencilIcon, TrashIcon} from '@heroicons/react/outline';
+import EmptyConnectors from '../../../../assets/redpanda/EmptyConnectors.svg';
+import {DeleteSecretRequest, Secret} from '../../../../protogen/redpanda/api/dataplane/v1alpha2/secret_pb';
+
+const {ToastContainer, toast} = createStandaloneToast();
+
+const CreateSecretButton = () => {
+    return (<Box style={{display: 'flex', marginBottom: '.5em'}}>
+        <Link to={'/rp-connect/secret/create'}><Button>Create secret</Button></Link>
```

Comment:
This is wrong when it comes to a11y, having a button instead of a link is semantically wrong. I think something like `<Button as={ReactRouterLink} to="...">` would be better

Reply:
fixed